:wq

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
This file name and path looks incorrect. :wq  ?

Reply:
My bad, good catch, thx a lot @dougsland